Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,29 @@
 import React from 'react';
```

Comment:
Get used to memorizing these patterns! They will definetly be useful to remember later.